README.md

```diff
@@ -140,6 +140,9 @@ packages:
 
     - package: dbt-labs/dbt_utils
       version: [">=1.0.0", "<2.0.0"]
+
+    - package: calogica/dbt_date
```

Comment:
Should we add more to the README about the ability for the customer to customize timezone data?

Reply:
yeah definitely - will do so in the the documentation phase of this project (there'll be a new PR and height task)